### ISSUE TITLE
NTR: Uroporphyrinogen synthase activity low in erythrocytes (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8354]
New term request Uroporphyrinogen synthase activity low in erythrocytes (elthat_130725145040): RBCs, red blood cells; URO synthase

**Comment by pnrobinson** [comment]
@MickeySegal
There are several related enzymes, could you specify which is meant?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK154652/ "Uroporphyrinogen III synthase (URO-synthase)... uroporphyrinogen III synthase EC 4.2.1.75"

I updated our finding name to specify this.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000514
- label: Reduced erythrocyte uroporphyrinogen III cosynthase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:3674403
- def: Activity or concentration of in the level of uroporphyrinogen III (EC 4.2.1.75) in erythrocytes below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8354
- synonyms:
